src/main/java/Updater/Updater.java

```diff
@@ -0,0 +1,43 @@
+package Updater;
+
+import java.io.IOException;
+import java.nio.file.*;
+import java.awt.Desktop;
+
+public class Updater {
+
+    public static void main(String[] args) {
+        String oldFilePath = "";
+        String newFilePath = "";
+        replaceUpdate(oldFilePath , newFilePath);
+    }
+
+    public static void replaceUpdate(String oldFilePath ,String newFilePath){
+        try {
+            //Replacing new files
+            Path oldPath = Path.of(oldFilePath);
+            Path newPath = Path.of(newFilePath);
+            Path copy = Files.copy(newPath, oldPath, StandardCopyOption.REPLACE_EXISTING);
+
+            if (Desktop.isDesktopSupported()) {
+                Desktop desktop = Desktop.getDesktop();
+                if (desktop.isSupported(Desktop.Action.OPEN)) {
+                    // Open the new file with the default application
+                    desktop.open(oldPath.toFile());
+                } else {
+                    System.out.println("Desktop doesn't support OPEN action.");
+                }
+            } else {
+                System.out.println("Desktop is not supported.");
+            }
```

Comment:
@kmr-rohit Please don't use Desktop. Simply use `ProcessBuilder` to launch the application.
You can see the below video to know how nice process builder handles the executable launch and exits just after starting the executable --- that is what we want 😀.

[Screencast from 2023-10-24 12-08-22.webm](https://github.com/SaptarshiSarkar12/Drifty/assets/105960032/37ecd5bb-eba3-446c-9c98-4f30d0bdb0fd)

Reply:
Used ProcessBuilder to launch Updater Application.

Reply:
Okay :+1: